### PR TITLE
Build specs with ENB

### LINKED ISSUE #479: Build specs with ENB

### PR DESCRIPTION
**Pull request by blond**
Resolved #479

### CODE REVIEW

**Comment by narqo** [comment]
### 1) According to Travis log, some tests run several times.

E.g.

```
08:14:29.018 - [specs] [desktop.specs/attach/attach.html] 
  attach
    value
  ✓ getVal should return actual input's value 
  ✓ should emit "change" event after input value changed 
  ✓ should properly extract file name 
  ✓ should properly extract file extension 
    clear
  ✓ should reset input's value 
  ✓ should emit "change" event after clear 
  ✓ should be called after "pointerclick" on "clear" elem 
    enable/disable
  ✓ should enable/disable button according to self "disabled" state 
  8 passing (142ms)
```

and

```
08:14:29.696 - [specs] [desktop.specs/attach/attach.html] 
  attach
    value
  ✓ getVal should return actual input's value 
  ✓ should emit "change" event after input value changed 
  ✓ should properly extract file name 
  ✓ should properly extract file extension (67ms)
    clear
  ✓ should reset input's value 
  ✓ should emit "change" event after clear 
  ✓ should be called after "pointerclick" on "clear" elem 
    enable/disable
  ✓ should enable/disable button according to self "disabled" state 
  8 passing (274ms)
```

and one more.

### 2) Some tests fail but ENB thinks that everything is ok

Example:

```
08:14:33.458 - [specs] [desktop.specs/popup/popup.html] 
  popup
    setTarget
  ✓ should throw "Error" if no target set 
  1) should consume x and y coordinates as targets
    attachment to body
    directions
      relative to position
      relative to owner
    scroll reactions
    setContent and redraw
    nested popups
    destructing
  1 passing (48ms)
  1 failing
  1) popup setTarget should consume x and y coordinates as targets:
      + expected - actual
      +15
      -21

      at file:////home/travis/build/bem/bem-components/desktop.specs/popup/_popup.spec.js:11030
```

File: package.json [hunk 1]
```diff
@@ -47,7 +48,7 @@
     "build": "enb make examples && enb make tests && enb make docs && enb make",
     "test": "npm run lint && npm run unit",
     "lint": "jshint-groups && jscs . && csscomb -vl .",
-    "unit": "bower-npm-install && bem make -w 10 desktop.specs touch-pad.specs touch-phone.specs",
+    "unit": "bower-npm-install && enb make specs",
```
Comment (tadatuta) [inline package.json]: здесь разве не достаточно `bower i && enb make specs`?
кстати, без `bower i` ведь не будут работать остальные цели про `build`, `test` и т.д., почему только здесь осталось?

**Comment by tadatuta** [comment]
@andrewblond how it's going?

**Comment by blond** [comment]
@tadatuta, to fix the (1) problem need resolve https://github.com/enb-make/enb/issues/114

**Comment by blond** [comment]
Fixed (2) problem

**Comment by tadatuta** [comment]
@narqo can we merge it as it is now? some tests will still run several times but we should get proper results.

**Comment by narqo** [comment]
> can we merge it as it is now [..]

What for? Any profits? 

_According to Travis log build for all specs takes same 3-5 minutes. To be honest I don't understand our profits now. Could we wait until our benefits would became more tangible (or at least ENB would completely satisfy our needs_?

**Comment by blond** [comment]
The bug (1) has been fixed!

**Comment by blond** [comment]
ping!

**Comment by narqo** [comment]
Looks good

**Pull request merged**